src/client/process.ts

```diff
@@ -134,22 +137,13 @@ export class PowerShellProcess {
         return this.sessionDetails
     }
 
-    public showConsole(preserveFocus: boolean) {
-        if (this.consoleTerminal) {
-            this.consoleTerminal.show(preserveFocus);
-        }
-    }
-
-    public async eval(line: string) {
-        if (this.consoleTerminal) {
-            this.consoleTerminal.sendText(line)
-        }
-    }
 
     public async scrollToBottom() {
-        this.consoleTerminal.show(false)
-        await utils.sleep(100)
-        await vscode.workspace.nvim.command("wincmd w")
+        if (this.config.integratedConsole.showOnStartup) {
+            this.consoleTerminal.show(!this.config.integratedConsole.focusConsoleOnExecute)
```

Comment:
remove this line, no focus = no autoscroll :/

Reply:
:(

Reply:
But, who cares if a hidden terminal autoscrolls? :D